operations/checklists/backup-artifact-checklist.md

```diff
@@ -0,0 +1,44 @@
+---
+- tags: 
+  - people
+  - operations
+  - checklist
+- title: "Checklist: Back up Artifact"
+- date: 2024-07-22
+- description: To make sure that project’s progress is aligned with the team plan, we collect artifacts every 3 months and save as record of what was done and can be referred to in case of any disputes or misunderstandings with the client.
```

Comment:
Ko nên mention "any disputes or misunderstandings with the client" là lý do chính. chỗ này có thể refer để onboarding hoặc có track record để trace back lại nếu cần.